src/main/java/net/snowflake/client/core/SFSessionProperty.java

```diff
@@ -112,7 +112,10 @@ public enum SFSessionProperty {
 
   HTTP_CLIENT_CONNECTION_TIMEOUT("HTTP_CLIENT_CONNECTION_TIMEOUT", false, Integer.class),
 
-  HTTP_CLIENT_SOCKET_TIMEOUT("HTTP_CLIENT_SOCKET_TIMEOUT", false, Integer.class);
+  HTTP_CLIENT_SOCKET_TIMEOUT("HTTP_CLIENT_SOCKET_TIMEOUT", false, Integer.class),
+
+  SKIP_TOKEN_FILE_PERMISSIONS_VERIFICATION(
```

Comment:
finally it is an env property so we can remove the entry here?

Reply:
True, removed.